### ISSUE TITLE
Outlook for MAC and applewebdata

### ISSUE THREAD

**Issue by fabramian** [gophish/gophish#344]
Hi,

I have an issue with a campaign Im testing. Basically the campaign works as intended except for one little issue I have with Mac and Outlook for Mac.

THe link is perfectly normal on PC however when receiving it in Outlook for Mac it adds some stuff to the link. Se example below:

On Mac: applewebdata://50F1C466-F0F2-4124-A1B6-752F1106F1B6/10.50.10.50:80?rid=fbce06e02e653f271f2a574905484a40f768b399a5f8e92edbf1b7a4ed708b16

On Windows: 10.50.10.50:80?rid=fbce06e02e653f271f2a574905484a40f768b399a5f8e92edbf1b7a4ed708b16

So in conclusion the link cannot be opened on the Mac. Any ideas why this is and how to work around it?
